### PR TITLE
Modernize code: Replace deprecated function calls

### PR DESCRIPTION
**Pull request by SmallJoker**

### CODE REVIEW

File: dragons/piloting.lua [hunk 1]
```diff
@@ -22,9 +22,9 @@ dmobs.dragon.ride = function(self, clicker)
 			object_detach(self, clicker, {x=1, y=0, z=1})
 			
 			if inv:room_for_item("main", "mobs:saddle") then
-					inv:add_item("main", "mobs:saddle")
+				inv:add_item("main", "mobs:saddle")
 			else
-					minetest.add_item(clicker.getpos(), "mobs:saddle")
```
Comment (SmallJoker) [inline dragons/piloting.lua:27]: Nice, this code never worked correctly.

**Comment by D00Med** [comment]
I'll merge this, but the mod is dead. 
I'm not maintaining it anymore and there are other errors

**Comment by SmallJoker** [comment]
Yet players still use it because it offers many different mobs. It's sad to see it's no longer maintained with a forum topic which is so active.

**Comment by D00Med** [comment]
That's irrelevant, if I'm not updating it I'm not updating it. 
Anyway, it's not maintained any more because I'm working on a replacement.

**Pull request merged**